File: api-gateway/src/main/java/com/goraceloty/apigateway/offers/entity/OfferChange.java
package com.goraceloty.apigateway.offers.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import java.time.LocalDateTime;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class OfferChange {
    private Long changeID;
    private Long offerID;
    private String committedChange;
    private String commitType;
    private LocalDateTime createdAt;
}

File: goraceloty-front-end/src/app/entity/OfferChange.ts
export class OfferChange {
  offerID: number;
  committedChange: string;
  commitType: string;
  createdAt: Date;

  constructor(offerID: number, committedChange: string, commitType: string, createdAt: Date) {
    this.offerID = offerID;
    this.committedChange = committedChange;
    this.commitType = commitType;
    this.createdAt = createdAt;
  }
}
